Category/categoryRouter.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+
+const Category = require('./Categories');
+
+const router = express.Router();
+
+
+router
+  .route('/')
+  .get((req, res) => {
+
+  Category.find()
+    .select({_id: 0, title: 1})
```

Comment:
Awesome! 👌 We can also do this `.select('-_id title')` since we're using Mongoose! 🦆